src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -3340,6 +3340,50 @@ public void WriteLargeKeyValue(bool formatted, bool skipValidation)
             WriteTooLargeHelper(options, key.Slice(0, 10_000_000 / 3), value.Slice(0, 10_000_000 / 3), noThrow: true);
         }
 
+        [Theory]
+        [InlineData("2019-04-24T14:50:17.0000000", "\"2019-04-24T14:50:17\"")]
+        [InlineData("2019-04-24T14:50:17.1000000", "\"2019-04-24T14:50:17.1\"")]
+        [InlineData("2019-04-24T14:50:17.0001000", "\"2019-04-24T14:50:17.0001\"")]
+        [InlineData("2019-04-24T14:50:17.0001500", "\"2019-04-24T14:50:17.00015\"")]
+        [InlineData("2019-04-24T14:50:17.0996100", "\"2019-04-24T14:50:17.09961\"")]
+        [InlineData("2019-04-24T14:50:17.0996141", "\"2019-04-24T14:50:17.0996141\"")]
+        [InlineData("2019-04-24T14:50:17.0000000Z", "\"2019-04-24T14:50:17Z\"")]
+        [InlineData("2019-04-24T14:50:17.1000000Z", "\"2019-04-24T14:50:17.1Z\"")]
+        [InlineData("2019-04-24T14:50:17.0001000Z", "\"2019-04-24T14:50:17.0001Z\"")]
+        [InlineData("2019-04-24T14:50:17.0001500Z", "\"2019-04-24T14:50:17.00015Z\"")]
+        [InlineData("2019-04-24T14:50:17.0996100Z", "\"2019-04-24T14:50:17.09961Z\"")]
+        [InlineData("2019-04-24T14:50:17.0996141Z", "\"2019-04-24T14:50:17.0996141Z\"")]
+        public void WriteDateTime_TrimsFractionCorrectly(string testStr, string expectedStr)
+        {
+            var options = new JsonWriterOptions { Indented = false, SkipValidation = false };
```

Comment:
The default JsonWriterOptions is false, false, so it isn't necessary to be passed in.

So, the following is fine:
```C#
using var jsonUtf8 = new Utf8JsonWriter(output);
```